diff --git a/etc/issue b/etc/issue
index ee00925..7b57cf2 100644
--- a/etc/issue
+++ b/etc/issue
@@ -1,2 +1,2 @@
-Ubuntu 18.04 LTS \n \l
+GalliumOS 3.0 \n \l
 
diff --git a/etc/issue.net b/etc/issue.net
index 91aeeb9..7b7c30c 100644
--- a/etc/issue.net
+++ b/etc/issue.net
@@ -1 +1 @@
-Ubuntu 18.04 LTS
+GalliumOS 3.0
diff --git a/etc/lsb-release b/etc/lsb-release
index 96aa62f..046e059 100644
--- a/etc/lsb-release
+++ b/etc/lsb-release
@@ -1,4 +1,4 @@
 DISTRIB_ID=Ubuntu
 DISTRIB_RELEASE=18.04
 DISTRIB_CODENAME=bionic
-DISTRIB_DESCRIPTION="Ubuntu 18.04 LTS"
+DISTRIB_DESCRIPTION="GalliumOS 3.0"
diff --git a/etc/os-release b/etc/os-release
index ee61310..eaf41e3 100644
--- a/etc/os-release
+++ b/etc/os-release
@@ -1,12 +1,11 @@
-NAME="Ubuntu"
-VERSION="18.04 LTS (Bionic Beaver)"
-ID=ubuntu
-ID_LIKE=debian
-PRETTY_NAME="Ubuntu 18.04 LTS"
-VERSION_ID="18.04"
-HOME_URL="https://www.ubuntu.com/"
-SUPPORT_URL="https://help.ubuntu.com/"
-BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
-PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
-VERSION_CODENAME=bionic
+NAME="GalliumOS"
+VERSION="3.0 (Bismuth)"
+ID=galliumos
+ID_LIKE="ubuntu debian"
+PRETTY_NAME="GalliumOS 3.0"
+VERSION_ID="3.0"
+HOME_URL="https://galliumos.org/"
+SUPPORT_URL="https://galliumos.org/"
+BUG_REPORT_URL="https://https://github.com/GalliumOS/galliumos-distro/issues"
+VERSION_CODENAME=bismuth
 UBUNTU_CODENAME=bionic
